lib/schema.coffee

```diff
@@ -152,3 +152,9 @@ module.exports =
 			description: 'wifi key'
 			type: 'string'
 			allowEmpty: true
+		connectivity:
+			description: 'network management software'
+			type: 'string'
+			allowEmpty: false
+			pattern: /^(networkmanager)|(connman)$/
```

Comment:
I believe you can express this better with a `revalidator` enum. See https://github.com/flatiron/revalidator#enum